.github/workflows/build_tiber.yml

```diff
@@ -9,21 +9,22 @@ on:
 
 permissions:
   contents: read
-  security-events: write
 
 env:
   BUILD_TYPE: "Release"
-  DOCKER_IMAGE_NAME: "docker.io/ovc/tiber-broadcast-suite"
+  DOCKER_IMAGE_NAME: "ghcr.io/openvisualcloud/intel-tiber-broadcast-suite/tiber-broadcast-suite"
```

Comment:
do we discussed where this image will be pushed at all ?

Reply:
This the only possibility - `ghcr.io` / `namespace` / `repository` / `artifact-name` : `tag`.
This ends up as a summary and as it is a free tier not the paid one, we does not see it anywhere other then as our artifacts: 
![image](https://github.com/user-attachments/assets/22cdd615-b640-46e4-9182-f78a1acbe19d)

https://github.com/OpenVisualCloud/Media-Communications-Mesh/actions/runs/11962570821